digitalforms-api/src/test/java/ca/bc/gov/open/pssg/rsbc/digitalforms/controller/QueryServiceControllerTests.java

```diff
@@ -1,86 +1,98 @@
+		status.setReviewCreatedYn("Y");
+		status.setReviewEndDtm("2018-06-29 00:00:00 -07:00");
+		status.setReviewStartDtm("2018-06-25 00:00:00 -07:00");
+		status.setReviewFormSubmittedYn("Y");
+		status.setSurnameNm("Smith"); 
+		
+		// good
+		when(service.getProhibitionStatus(NOTICE_NUMBER)).thenReturn(
+				new VipsProhibitionStatusResponse(status, DigitalFormsConstants.ORDS_SUCCESS_CD, DigitalFormsConstants.JSON_RESPONSE_SUCCESS));
+		
+		// not found
+		when(service.getProhibitionStatus(NOTICE_NUMBER_NOT_FOUND)).thenReturn(
+				new VipsProhibitionStatusResponse(status, DigitalFormsConstants.ORDS_FAILURE_CD, DigitalFormsConstants.JSON_RESPONSE_FAIL));
+		
+		
 	}
 
-	// Test irpGet for 200 returned on success.
-	// TODO - update when fully functional
+	@DisplayName("Get success status code - QueryServiceController")
 	@Test
-	void irpGetReturns200() {
-//		ResponseEntity<JSONResponse<QueryInfoResponse>> resp = controller.irpGet(IRP_TEST_ID);
-//		Assertions.assertEquals(HttpStatus.OK, resp.getStatusCode());
-		
-		Assertions.assertTrue(true);
+	void getReturns200() throws Exception {
+		ResponseEntity<JSONResponse<ProhibitionStatusResponse>> resp = controller.getProhibitionInfo(NOTICE_NUMBER);
+		Assertions.assertEquals(HttpStatus.OK, resp.getStatusCode());		
 	}
 
-	// Test irpGet for proper JSON reponse on success.
-	// TODO - update when fully functional
+	@DisplayName("Get response value success - QueryServiceController")
 	@Test
-	void irpGetReturnsSuccess() {
-//		ResponseEntity<JSONResponse<QueryInfoResponse>> resp = controller.irpGet(1L);
-//		Assertions.assertEquals(JSON_RESPONSE_GOOD, resp.getBody().getData().getIRPInfo().getCancelledYN());
-		
-		Assertions.assertTrue(true);
+	void getReturnsSuccess() throws Exception {
+		ResponseEntity<JSONResponse<ProhibitionStatusResponse>> resp = controller.getProhibitionInfo(NOTICE_NUMBER);
+		Assertions.assertEquals(ORIGINAL_CAUSE, resp.getBody().getData().getStatus().getOriginalCause());
 	}
 
-	// Test irpGet for IRP not found.
-	// TODO - update when fully functional
+	@DisplayName("Get fail status code - QueryServiceController")
 	@Test
-	void irpGetReturnsNotFound() {
-		// test for not found
-		
-		Assertions.assertTrue(true);
+	void getReturns404() throws Exception {
+		ResponseEntity<JSONResponse<ProhibitionStatusResponse>> resp = controller.getProhibitionInfo(NOTICE_NUMBER_NOT_FOUND);
+		Assertions.assertEquals(HttpStatus.NOT_FOUND, resp.getStatusCode());	
 	}
 
-	// Test irpGet for exception state.
-	// TODO - update when fully functional
+	@DisplayName("Get response error object status code - QueryServiceController")
 	@Test
-	void irpGetReturnException() {
-		// test for exception 
-		
-		Assertions.assertTrue(true);
+	void getReturnFailHttpStatus() throws Exception { 
+		ResponseEntity<JSONResponse<ProhibitionStatusResponse>> resp = controller.getProhibitionInfo(NOTICE_NUMBER_NOT_FOUND);
+		Assertions.assertEquals(404, resp.getBody().getError().getHttpStatus());
 	}
```

Comment:
could we combine both the assertions in one function? since its essentially the same test with multiple assertions

Reply:
Sure we can. Done. Thanks.